feature-libs/organization/administration/components/shared/form/form.component.ts

```diff
@@ -81,9 +88,21 @@ export class FormComponent<T> {
     this.i18n = this.i18nRoot + (item ? '.edit' : '.create');
   }
 
+  protected setIsInEdit(): void {
```

Comment:
the logic is so compact now, what about dropping this method and moving it to ngOnInit directly?

Reply:
agree